src/actions/sets.ts

```diff
@@ -9,8 +9,9 @@ import { NotificationLevel } from "../models/notification";
 
 export enum GraphSetActionType {
 	INIT = "INIT_SETS",
-	INIT_PRESETS = "INIT_PRESETS",
 	SET_SET_PRESET_LATEST = "SET_SET_PRESET_LATEST",
+	INIT_PRESETS = "INIT_PRESETS",
```

Comment:
For consistency when debugging actions / state would name this `SET_INIT_PRESETS`